pages/hud.coffee

```diff
@@ -10,6 +28,38 @@ handlers =
     document.getElementById("hud").classList.add "vimiumUIComponentHidden"
     document.getElementById("hud").classList.remove "vimiumUIComponentVisible"
 
+  showFindMode: (data) ->
+    hud = document.getElementById "hud"
+    hud.innerText = "/"
+
+    inputElement = document.createElement "span"
+    inputElement.contentEditable = "plaintext-only"
+    inputElement.textContent = data.text
+    inputElement.id = "hud-find-input"
+    hud.appendChild inputElement
+
+    inputElement.addEventListener "input", (event) ->
+      # Replace \u00A0 (&nbsp;) with a normal space.
+      UIComponentServer.postMessage {name: "search", query: inputElement.textContent.replace "\u00A0", " "}
```

Comment:
Again, this can't be in `FindMode` (at least yet) because they're in different frames.